pkg/gitprovider/git_provider.go

```diff
@@ -102,6 +102,9 @@ func (a *AbstractGitProvider) parseStaticGitContext(repoUrl string) (*StaticGitC
 	branchPath := strings.Join(parts[2:], "/")
 	if branchPath != "" {
 		repo.Path = &branchPath
+	} else {
```

Comment:
Is there an error that this solves? I don't think this is necessary and could lead to problems with repos that have something else listed as the default branch.

Reply:
Yes, you are right. I will remove it.